### PR TITLE
Added css for background-attachment scroll for other devices

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Advanced Column block displays a blurry background image.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Checked by inspecting whether the CSS work for other devices or not.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/columns/style.scss [hunk 1]
```diff
@@ -219,6 +219,14 @@
 	}
 }
 
+@media (min-width: 768px) and (max-width: 1024px) {
+  
+  .uagb-columns__wrap.uagb-columns__background-image{
+  		background-attachment: scroll!important;
```
Comment (vrundakansara) [inline src/blocks/columns/style.scss:225]: We are not using !important.

Comment (AbhijitNage123) [reply src/blocks/columns/style.scss]: Added below CSS
`@media (min-width: 768px) and (max-width: 1024px) {
  
  .wp-block-uagb-columns.uagb-columns__wrap.uagb-columns__background-image{
  		background-attachment: scroll;	
  	}
  
}`

**Comment by vrundakansara** [comment]
Add in changelog too.

**Comment by vrundakansara** [comment]
@AbhijitNage123 - Do you think we need the same change in `Section` block too?

**Comment by AbhijitNage123** [comment]
Right @vrunda ma'am we need to fix same for the section.

**Pull request merged**